packages/react-dom/src/client/ReactDOMLegacy.js

```diff
@@ -250,6 +259,15 @@ export function render(
   container: Container,
   callback: ?Function,
 ) {
+  if (__DEV__) {
+    console.error(
+      'ReactDOM.render is no longer supported in React 18. Use createRoot ' +
```

Comment:
Maybe use  "ReactDOM.render is deprecated in React 18" instead for consistent messaging? "not supported" sounds like it's no longer working. Unless the other deprecation warnings also use "not supported" instead of "deprecated".